readme/LinuxBuild.md

```diff
@@ -6,6 +6,8 @@ Please create pull requests (or file issues) if you have more precise dependenci
 
 Lua is needed for plugins, if it is missing, the plugins will be disabled.
 
+The installation instructions don't assume any specific build tool (other than CMake), but they do require make, ninja, or another supported CMake generator. It is required that such a tool is installed in order to build xournalpp.
```

Comment:
Very minor nitpick, but could you break this line into multiple lines with less than 80 (or 90) characters to be consistent with the rest of the file?